MobileCenter/MobileCenter.xcodeproj/project.pbxproj

```diff
@@ -1016,11 +1016,19 @@
 				APPLICATION_EXTENSION_API_ONLY = NO;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				FRAMEWORK_SEARCH_PATHS = "\"$(SRCROOT)/../Vendor\"/**";
-				HEADER_SEARCH_PATHS = "\"$(SRCROOT)/../Vendor\"/**";
 				INFOPLIST_FILE = MobileCenterTests/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "\"$(SRCROOT)/../Vendor\"/**";
+				OTHER_CFLAGS = (
+					"$(inherited)",
+					"-iframework",
+					"\"$(SRCROOT)/../Vendor/OCHamcrest\"",
+					"-iframework",
+					"\"$(SRCROOT)/../Vendor/OHHTTPStubs\"",
+					"-isystem",
+					"\"$(SRCROOT)/../Vendor/OCMock\"",
+				);
```

Comment:
Please move those to the xcconfig-file. We want to utilize thos as much as possible.  The entry in the project file will then only just be:
```
OTHER_CFLAGS = (
					"$(inherited)"
);```